### ISSUE TITLE
Missing Servers in Dashboard (webui version: 24b30ef)

### ISSUE THREAD

**Issue by ghost** [hexparrot/mineos#127]
Hello.  I recently updated mineos and now none of my servers (offline or online) appear in the dashboard.  I tried creating a couple servers (one from a backup, the other fresh) just as a test and those do not show either even though I receive a successful message.

Based on another thread, I ran the following that may help.
me@host:/usr/games/minecraft$ ./mineos_console.py -d /var/games/minecraft ping_debug
Miami_1.8: current (1.8) - 1.8
MyServer: unknown (0.0) - unknown
Traceback (most recent call last):
  File "./mineos_console.py", line 158, in <module>
    prop_ = getattr(mc(i, None, args.base_directory), args.cmd)
  File "/usr/games/minecraft/mineos.py", line 1013, in ping_debug
    self.server_milestone,
  File "/usr/games/minecraft/mineos.py", line 980, in server_milestone
    jar_file = self.valid_filename(self.profile_config[self.profile:'run_as'])
  File "/usr/games/minecraft/conf_reader.py", line 92, in __getitem__
    raise TypeError('First argument must be string not %s' % type(option.start))
TypeError: First argument must be string not <type 'NoneType'>

All my servers appear to still be there under /var/games/minecraft/servers

Here is a little about my system, it is just my desktop and used for home use.  Not sure if others have received a similar experience.

me@host:~$ cat /etc/os-release 
NAME="Ubuntu"
VERSION="14.04.3 LTS, Trusty Tahr"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.04.3 LTS"
VERSION_ID="14.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
me@host:~$ uname -r
3.13.0-62-generic
me@host:~$ python --version
Python 2.7.6
me@host:~$ python3 --version
Python 3.4.0

**Comment by HameedSilvertoe** [comment]
I seem to be having the same or similar issue.  I just updated from the python version because I did a server shift this weekend.  As long as I only have one world installed (I'm importing using the webui) everything shows up and works well.  As soon as I import a second world I lose the first server view.  Sometimes I can see the second imported world and sometimes I can't see any world but the servers running indicates that one or both are running (depending on whether I start the second or not).
